scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -17,7 +17,9 @@ Param(
     [AllowNull()]
     [string]$MesosWorkDir,
     [AllowNull()]
-    [string]$customAttrs
+    [string]$customAttrs,
+    [AllowNull()]
+    [string]$DcosVersion = "1.11.2"
```

Comment:
I would set the default to empty string. Then if the string is empty, I would do to the master node to get the version. Otherwise use provided value.